pkg/clustermesh/common/metrics.go

```diff
@@ -49,6 +51,13 @@ func MetricsProvider(subsystem string) func() Metrics {
 				Name:      "remote_cluster_failures",
 				Help:      "The total number of failures related to the remote cluster",
 			}, []string{metrics.LabelSourceCluster, metrics.LabelSourceNodeName, metrics.LabelTargetCluster}),
+
+			TotalCacheRevocations: metric.NewGaugeVec(metric.GaugeOpts{
+				Namespace: metrics.Namespace,
+				Subsystem: subsystem,
+				Name:      "remote_cluster_cache_revocations",
+				Help:      "The total number of cache revocations related to the remote cluster",
+			}, []string{metrics.LabelSourceCluster, metrics.LabelTargetCluster, metrics.LabelMode}),
```

Comment:
The source cluster label is going to get removed soon ~ for those case (https://github.com/cilium/cilium/pull/41976), so we can probably not add the source cluster label here too :sweat_smile:.

```suggestion
			}, []string{metrics.LabelTargetCluster, metrics.LabelMode}),
```

Also could you add this new metrics in the documentation in `Documentation/observability/metrics.rst` for all relevant entries (agent, operator, kvstoremesh IIUC) :pray: